project/src/lib/animations/simple.ts

```
@@ -1,65 +1,43 @@
     },
     transition: {
-        opacity: { delay: 0.2 },
-        x: { delay: 0.2 },
+        opacity: { delay: options?.delay ?? 0.2 },
+        x: { delay: options?.delay ?? 0.2 },
         y: {
-            delay: 0.6,
-            duration: 5,
+            delay: options?.bounceDelay ?? 0.6,
+            duration: options?.bounceDuration ?? 5,
             repeat: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)